src/SourceBuild/content/test/scenario-tests.proj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.Build.Traversal">
```

Comment:
So is the intention here to treat the scenario tests repo as "part" of the VMR, rather than a separate component that is lives in the VMR, like runtime, installer, etc.

Reply:
I want to warn on directly referencing projects from an inner repository. `ProjectReference` items transfer additional state. Here this means that additional state from the VMR build infrastructure might be passed into the scenario-tests projects within the scenario-tests repo.

This might not be an issue today but this crosses the repository boundary which I would prefer to avoid when possible. When we discussed this a few weeks ago, I was under the assumption that we could just call into the scenario-tests build entrypoint and pass the `-test` action and additional msbuild properties in by leveraging the already existing `repo-projects/scenario-tests.proj` project.

Reply:
Fixed in https://github.com/dotnet/installer/pull/19222/commits/09b93d4d4ac9a4d396086c5e96dc7b15a52359d8